manifests/server/grant_role.pp

```diff
@@ -32,7 +32,7 @@
 
   postgresql_psql { "grant_role:${name}":
     command          => $command,
-    unless           => "SELECT 1 WHERE pg_has_role('${role}', '${group}', 'MEMBER') ${unless_comp} true",
+    unless           => "SELECT t.count FROM (SELECT count(*) FROM pg_roles AS r_role JOIN pg_auth_members AS am ON (r_role.oid = am.member) JOIN pg_roles AS r_group ON (r_group.oid = am.roleid) WHERE r_group.rolname = '${group}' AND r_role.rolname = '${role}') AS t WHERE t.count ${unless_comp} 1",
```

Comment:
Something like `SELECT 1 WHERE EXISTS(...) ${unless_comp} true` as before would look nicer in here.

No need for parentheses on ON clauses.